test/expectations/folding_ranges/block_comments_surround.exp.json

```diff
@@ -0,0 +1,27 @@
+{
+  "result": [
+    {
+      "startLine": 2,
+      "endLine": 4,
+      "kind": "region"
+    },
+    {
+      "startLine": 2,
+      "endLine": 4,
+      "kind": "region"
+    },
```

Comment:
Yes, there are some situations where we return duplicate ranges. It causes no problem on the client side, so we ended up not putting any code in to prevent the duplicates, which would complicate our implementation.